miner/worker.go

```diff
@@ -1449,13 +1478,17 @@ func (w *worker) commitNewWork(interrupt *int32, noempty bool, timestamp int64)
 			return
 		}
 	}
+	l2CommitNewWorkPrioritizedTxCommitTimer.UpdateSince(prioritizedTxStart)
```

Comment:
can also add metrics to track `NewTransactionsByPriceAndNonce`, because for `commitTransactions`, there are timers inside.

Reply:
[af554cd](https://github.com/scroll-tech/go-ethereum/pull/700/commits/af554cd174359d1a71182c4c9e7af32cc07ed538)